### ISSUE TITLE
Excessive output during tests when geometry files can't be found

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-core#958]
I get a bunch of the following messages during tests (not just once, but many times, it seems):

```
ModelVisualizer couldn't find file 'block.vtp'; tried
 /Users/chris/repos/opensim/2/build/Applications/Scale/test/block.vtp
 /Users/chris/repos/opensim/2/build/Applications/Scale/test/Geometry/block.vtp
 /usr/local/OpenSim/Geometry/block.vtp
    Set environment variable OPENSIM_HOME to search $OPENSIM_HOME/Geometry.
```

This seems to be a new thing, probably related to how the caching of geometry occurs?

**Comment by aymanhab** [comment]
@chrisdembia This is a side effect of moving the loading/caching of Geometry files during initialization/finalizeFromProperties vs. lazily on first draw which would cause the messages to show only if we're visualizing. We can remove or silence these for now or have a flag to Debug/verbose visualization. Let me know what you think.

**Comment by chrisdembia** [comment]
I'm not sure. For the mean time we could also just give the message once and stop trying to load the file if we've already tried once. I think you were doing this before with that bool flag (that I had questioned haha).

**Comment by aseth1** [comment]
> I get a bunch of the following messages during tests (not just once, but many times, it seems):

Seems like we have to figure out why `finalizeFromProperties` is being invoked multiple times. Also, if the Geometry is up-to-date with its properties it would be fine not to reload the Geometry.

**Comment by chrisdembia** [comment]
Looking into it further, the issue may not be repeated calls to finalizeFromProperties(), but rather the fact that there many geometry files in most models, and each one gives a warning.

**Comment by tkuchida** [comment]
Fixed by #1120
